### PR TITLE
Made 500 sentences for the Dutch language.

### PR DESCRIPTION
**Pull request by pcmill**
The first 500 sentences to get the Dutch language going for the project. Closes issue #1340.

### CODE REVIEW

File: server/data/nl/technologie.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+De laptop was erg fijn maar wel duur.
+De nieuwste gadget was niet voor iedereen weggelegd.
+Door de muziek te streamen had hij geen kabels nodig.
+Uit het geweer kwam veel rook, maar geen kogel.
+Voor gepolariseerd licht ligt het einde van de vector op het oppervlak van de bol.
+Algoritmes zijn simpel om te implementeren.
+Het weegt vijf kilo.
+H2O is de chemische formule voor water.
+Het nummer pi is ongeveer drie punt vier één vijf.
+De CIA is een geheime spionnen organisatie.
```
Comment (sroet) [inline server/data/nl/technologie.txt]: CIA is een afkorting

File: server/data/nl/technologie.txt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+De prijzen voor zonnepanelen zijn de afgelopen jaren enorm gedaald.
```
Comment (sroet) [inline server/data/nl/technologie.txt]: "prijs van" lijkt me logischer

File: server/data/nl/zakelijk.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
+De mensen waar ik voor werk willen je op tijdelijk basis inhuren.
+Ze willen je ontmoeten om de zaken te bespreken.
+Ik hoop dat deze relatie veel oplevert.
+Ik weet niet zeker of we willen samenwerken met zo'n organisatie.
+Misschien kan een van je collega's wat ophelderen over de zaak.
+We hebben een uitgebreid selectie proces voor nieuwe medewerkers.
+Niemand nam op, dus ik ben zelf even langs gegaan.
+Geld heeft niet mijn interesse, macht wel!
+Samen liepen ze door de winkel heen.
```
Comment (sroet) [inline server/data/nl/zakelijk.txt]: "heen" is onlogisch hier

File: server/data/nl/zakelijk.txt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+De verkoper was net als de rest niet betrouwbaar.
+Het hele kantoor was stilgevallen.
+De eisen voor deze baan zijn vrij hoog.
+Om een of andere reden mochten we niet naar binnen.
+Hoe weet ik zeker dat je mijn naam uit het verhaal houdt?
+De leiding hebben kan heel vermoeiend zijn.
+Hij besloot dat het niet genoeg was.
+Mijn loon wordt pas volgende week gestort.
+Samen kunnen we de wereld aan!
+De voelden aan dat de verkoper ze voorgelogen had.
```
Comment (sroet) [inline server/data/nl/zakelijk.txt]: "De" => "Ze"

File: server/data/nl/zakelijk.txt [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+Mijn zakenreis naar Brussel was een ramp.
+Als je korting wil krijgen moet je een ID kaart kunnen tonen.
```
Comment (sroet) [inline server/data/nl/zakelijk.txt]: ID is een afkorting voor identiteitskaart

File: server/data/nl/zakelijk.txt [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+Verlaat mijn winkel.
+De CEO van dat Japanse bedrijf is afgetreden.
+Kan je die papieren in de lade leggen?
+Deze verklaring is geheel vals.
+De vergadering liep tien minuten uit.
+Het hele bedrijf moest agile gaan werken.
+De CV van de kandidaat was zeer indrukwekkend.
```
Comment (sroet) [inline server/data/nl/zakelijk.txt]: CV is afkorting voor curriculum vitae

**Comment by sroet** [comment]
Looks good to me! Thank you again for putting this together. 
@mikehenrty are you the right person to ping for a merge?

**Comment by sroet** [comment]
@Gregoor I heard that you are (for now) the one to ping for a merge

**Comment by Gregoor** [comment]
Yes! Thanks for taking note of that, Michael & I forgot to talk about this before he left but I think he'd want me to not stop y'all from getting new sentences in 🙂 Thanks for the sentences & reviewing!

**Pull request merged**